docker-compose.dev.yml

```diff
@@ -15,7 +15,7 @@ services:
       - "8010:80"
     environment:
       DEV_RUN: "true"
-      TEST_SERVICE_TOKEN:
+      TEST_SERVICE_TOKEN: "${TEST_SERVICE_TOKEN}"
```

Comment:
Does this change do anything? Doesn't an empty env variable means that it will pick it from the env?